packages/protoplugin-test/src/import_extension.test.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { createEcmaScriptPlugin, Schema } from "@bufbuild/protoplugin";
+import type { GeneratedFile } from "@bufbuild/protoplugin/ecmascript";
+import { CodeGeneratorRequest } from "@bufbuild/protobuf";
+
+/**
+ * Generates a single file using the plugin framework and the given print function,
+ * passing the given options to the plugin.
+ * The returned string array represents the generated file content created using printFn.
+ */
+function generate(
+  printFn: (f: GeneratedFile, schema: Schema) => void,
+  options: string[]
+): string[] {
+  const req = new CodeGeneratorRequest({
```

Comment:
I'm very wary of using it everywhere because it makes the test depend on large parts of @bufbuild/protoplugin and @bufbuild/protobuf. Ideally, unit tests test only small units of code.

Removed it from two other locations in https://github.com/bufbuild/protobuf-es/pull/317/commits/fff972562e2ff408219ea8aa2953f9eaacf40985. The few tests that need a `Desc*` now just get one via getDescriptorSet().